### DISCUSSION TITLE
Collecting Requirements/Ideas/Features for v1.10.0

### DISCUSSION THREAD

**Discussion by OrlinVasilev** [vmware-tanzu/velero#5003] [General discussions]
Hello Community!

Velero v1.9 is coming within the next weeks, and before we start working on v1.10, we’d love to brainstorm with you on ideas and requirements for the feature list of this next release!

Please review what we have already laid out in [the v1.10 roadmap](https://github.com/vmware-tanzu/velero/wiki/1.10-Roadmap) and share your thoughts/ideas/requirements/feedback here so we can all work together on building out the next version of Velero.

**Comment by shirishbhalekar** [comment]
What we are able to perform using command line ,all ability we can add to REST API access of velero.

**Reply by reasonerjt** [reply]
I'm a little confused there's no API server for velero.

**Reply by OrlinVasilev** [reply]
@shirishbhalekar  do you mean you would like to have API implemented on the server side?

**Reply by shirishbhalekar** [reply]
Yes Exactly AS currently what we can do using Velero command line after installation every thing we should do by Using RestAPI.

**Comment by phuongatemc** [comment]
We would like to finish off the plugin version refactor and add version 2 for BackupItemAction with Context for backup timeout.

**Reply by reasonerjt** [reply]
Thanks @phuongatemc we'll put this into the roadmap.

**Comment by jsanda** [comment]
Any chance https://github.com/vmware-tanzu/velero/issues/4962 could be considered for 1.10? The project I work on, https://k8ssandra.io/, needs to support hostPath volumes both for dev/testing as well as for some production scenarios. Unfortunately without support for hostPath I cannot really move forward with integrating Velero into the project. I am be happy to spend time working on it, but I would need someone I could bounce questions off of.

**Comment by maikel-hoorn** [comment]
Restic recently released version 0.14.0 (2022-08-25).

The feature that we are interested in is https://github.com/restic/restic/pull/3773: Optimize memory usage for directories with many files

Any chance this will be included in 1.10.0?

**Comment by OrlinVasilev** [comment]
As 1.10 is out https://github.com/vmware-tanzu/velero/releases/tag/v1.10.0 we will close this discussion!
